Initialize Keptn Project Pages (#143)
keptn/community: mentorship/gsoc/2022/projects/gitops-deployment-tools-integration/README.md

GSoC 2022 - Integration for GitOps-based Deployment Tools (e.g. ArgoCD/Flux)
==========

To be able to use GitOps tools in combination with Keptn.
The main goal of this project is to find a proper way how the tools could interact with Keptn and to build an integration for such tools.

> Deployment tools can be integrated via the helm service or via the job-executor service.

Reviewer: ```suggestion
Deployment tools can be integrated via the helm-service or via the job-executor service.
```

Currently, there’s no way to integrate GitOps Tools in Keptn.

> Provide a way such that it could be integrated into Keptn return deployment states after the state has been synced.

Reviewer: ```suggestion
Provide a way such that such tools can be integrated into Keptn return deployment states after the state has been synced.
```
